packages/react-vapor/src/components/datePicker/DatePickerSelectors.ts

```diff
@@ -11,11 +13,21 @@ const getDatePicker = (state: IReactVaporState, props: {id: string}): IDatePicke
     return (datePickers && datePickers[0]) || null;
 };
 
-const getDatePickerLimits = (state: IReactVaporState, props: {id: string}): [Date, Date?] => {
+const getLimits = (state: IReactVaporState, props: {id: string}): [Date, Date?] => {
     const picker = DatePickerSelectors.getDatePicker(state, props);
     return picker ? [picker.appliedLowerLimit, picker.appliedUpperLimit] : [null, null];
 };
 
+const getDatePicker = createSelector(

```

Comment:
this one is not necessary because, the createSelector is used to memorize the result of the combinaison of multiple selector.

In your case, you have 1 selector and you return the value returned by your select. You value is already memorize by the mapStateToProps.

Same for getDatePickerLimits under